core/interfaces.go

```diff
@@ -173,6 +173,11 @@ type Broadcaster interface {
 	Broadcast(context.Context, Duty, PubKey, SignedData) error
 }
 
+// Tracker sends core component events for further analysis and instrumentation.
+type Tracker interface {
+	SendEvent(context.Context, Duty, string, PubKey, error, ParSignedData) error
```

Comment:
We'll need interface with methods for each event type, cannot use a common interface for all.

Reply:
error should be last, and do not return an error as well

Reply:
Why do we need methods for each type?
Since tracker events mostly takes duty, component name and pubkey for failed duty analysis. And it takes ParSignedData from parsigdb and parsigex components to analyse participation.

Reply:
Understood!